kbchat/team.go

```diff
@@ -27,7 +27,7 @@ func (a *API) ListMembersOfTeam(teamName string) (res keybase1.TeamMembersDetail
 	apiInput := fmt.Sprintf(`{"method": "list-team-memberships", "params": {"options": {"team": "%s"}}}`, teamName)
 	cmd := a.runOpts.Command("team", "api")
 	cmd.Stdin = strings.NewReader(apiInput)
-	bytes, err := cmd.CombinedOutput()
+	bytes, err := cmd.Output()
 	if err != nil {
 		return res, APIError{err}
```

Comment:
can you read from stderr on error and log?

Reply:
Should I log stderr if it isn't empty, or just if there is an `err`?

Reply:
always logging is fine, thanks!

Reply:
I log stderr for the methods that I modified, however this isn't don't elsewhere and probably should be.